package orwell.com;

// Base message used for communication, it contains an header defining the type of message
// And another message serialized in a byte array
message base_message {
	required string message_type = 1;
	required bytes serialized_message = 2;
}

// This message is used to handle a new client's login
message login_message {
	required string client_id = 1;
	required string wished_robot_type = 2;
}

message sample_message {
	required string login = 1;
}

